CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 5.0.0
+* New or changed parameters:
+    * Add server_ip for configuring the listen IP (puppetserver only)
+* Other features:
+    * Support puppetserver 2.x
+* Other changes and fixes:
+    * Set hiera_config correctly for Puppet 4.x
```

Comment:
@ekohl @domcleal I think we should open a 4.3.x branch and cherry-pick that hiera-config fix, what so you think?